service/routes.go

```diff
@@ -0,0 +1,21 @@
+package service
+
+import (
+	"github.com/gin-gonic/gin"
+)
+
+func Routes(router *gin.Engine) *gin.Engine {
+	router.GET("/ping", Ping)
+	router.GET("/book/:id", BookInfo)
+	router.PUT("/book", AddBook)
+	router.POST("/book", UpdateBookTitle)
+	router.DELETE("/book/:id", DeleteBook)
```

Comment:
You can have a group of routes with a shared prefix together, see how we did it in `go-searcher`